Composer/packages/client/src/components/ManageService/ManageService.tsx

```diff
@@ -0,0 +1,693 @@
+      props.onDismiss();
+    } else {
+      hasAuth();
+    }
+  };
+
+  const iconStyles = { marginRight: '8px' };
+  const onRenderOption = (option) => {
+    return (
+      <div>
+        {option.data && option.data.icon && (
+          <Icon aria-hidden="true" iconName={option.data.icon} style={iconStyles} title={option.data.icon} />
+        )}
+        <span>{option.text}</span>
+      </div>
+    );
+  };
+
+  const renderPageOne = () => {
+    return (
+      <div>
+        <div css={dialogBodyStyles}>
+          <p css={{ marginTop: 0 }}>
+            {formatMessage(
+              'Choose from existing {service} keys, create a new {service} resource, or generate a request to handoff to your Azure admin. ',
+              { service: props.serviceName }
+            )}
+            {props.learnMore ? (
+              <Link href={props.learnMore} target={'_blank'}>
+                {formatMessage('Learn more')}
+              </Link>
+            ) : null}
+          </p>
+          <div css={mainElementStyle}>
+            <ChoiceGroup options={actionOptions} selectedKey={nextAction} onChange={onChangeAction} />
+          </div>
+        </div>
+        <DialogFooter>
+          <PrimaryButton disabled={loading !== undefined} text={formatMessage('Next')} onClick={performNextAction} />
+          <DefaultButton
+            disabled={loading !== undefined || showAuthDialog}
+            text={formatMessage('Cancel')}
+            onClick={props.onDismiss}
+          />
+        </DialogFooter>
+      </div>
+    );
+  };
+
+  const renderPageChoose = () => {
+    return (
+      <div>
+        <div css={dialogBodyStyles}>
+          <p css={{ marginTop: 0 }}>
+            {formatMessage(
+              'Choose from existing {service} keys, create a new {service} resource, or generate a request to handoff to your Azure admin. ',
+              { service: props.serviceName }
+            )}
+            {props.learnMore ? (
+              <Link href={props.learnMore} target={'_blank'}>
+                {formatMessage('Learn more')}
+              </Link>
+            ) : null}
+          </p>
+          <div css={mainElementStyle}>
+            <Dropdown
+              disabled={!(availableSubscriptions?.length > 0)}
+              label={formatMessage('Select subscription')}
+              options={
+                availableSubscriptions
+                  ?.filter((p) => p.subscriptionId && p.displayName)
+                  .map((p) => {
+                    return { key: p.subscriptionId ?? '', text: p.displayName ?? 'Unnamed' };
+                  }) ?? []
+              }
+              placeholder={formatMessage('Select one')}
+              selectedKey={subscriptionId}
+              styles={dropdownStyles}
+              onChange={onChangeSubscription}
+            />
+          </div>
+          <div>
+            {noKeys && subscriptionId && (
+              <span style={{ color: 'rgb(161, 159, 157)' }}>
```

Comment:
Can we get this color from an existing UI palette?

Reply:
yup!